outlines/1.3-migration

```diff
@@ -0,0 +1,31 @@
+1. Breaking changes (as we'd normally do this)
+
+2. Recommended changes : Modules [link to app structure guide]
+  1. Applications: Move all files to imports, initialize in one spot, link via `import`
+  2. Import *all* symbols from packages, including meteor core packages.
```

Comment:
I brought it up before and I think @avital considered this to be more something to think about for 1.4, but I think it wouldn't be that much work. I'd be willing to just put in the leg work for documenting and exporting this stuff, as long as we know there are no technical blockers.

Everything I've heard from the community indicates that people consider it a best practice to start importing everything, including Meteor packages, so it would be nice to give people the tools to do so.